src/Debug.h

```diff
@@ -184,8 +184,8 @@ void _db_rotate_log(void);
 class Raw
 {
 public:
-    Raw(const char *label, const char *data, const size_t size):
-        level(-1), label_(label), data_(data), size_(size), useHex_(false), useGap_(true) {}
+    Raw(const char *label, const char *data, const size_t size, const size_t limit = 0):
```

Comment:
Please remove the limit from the constructor arguments. Raw uses dedicated methods for configuration options like that (e.g., hex(), gap(), and minLevel()). Please add such a method if at least one caller needs it. I suggest calling it atMost(), avoiding an overly generic "limit".

Reply:
done in dc3814d